docs/source/migrationGuide2.0.rst

```diff
@@ -1,34 +1,69 @@
-|                                                          |                                                                                                                                   |
-|                                                          |  image = sitk.ReadImage(file_name)                                                                                                |
-|                                                          |  grayscale_image = sitk.Cast(sitk.RescaleIntensity(0.299*sitk.VectorIndexSelectionCast(image,0,sitk.sitkFloat32) +                |
-|  image = sitk.ReadImage(file_name)                       |                                                    0.587*sitk.VectorIndexSelectionCast(image,1,sitk.sitkFloat32) +                |
-|  grayscale_image = sitk.VectorIndexSelectionCast(image,0)|                                                    0.114*sitk.VectorIndexSelectionCast(image,0,sitk.sitkFloat32)), sitk.sitkUInt8)|
-+----------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------+
+Filter's Execute Method
++++++++++++++++++++++++
 
+The Filters' `Execute` methods with filter parameters as arguments is removed. A filter object's parameters are only set via the SetXYZ methods. For example the `STAPLEImageFilter class <https://simpleitk.org/doxygen/latest/html/classitk_1_1simple_1_1STAPLEImageFilter.html>`__:
 
-Binary Morphology
++-----------------------------------------------------------+---------------------------------------------------------+
+| 1.x                                                       | 2.x                                                     |
++-----------------------------------------------------------+---------------------------------------------------------+
+| .. code-block:: python                                    | .. code-block:: python                                  |
+|                                                           |                                                         |
+|  staple_filter = sitk.STAPLEImageFilter()                 |  staple_filter = sitk.STAPLEImageFilter()               |
+|                                                           |  staple_filter.ConfidenceWeight(confidenceWeight)       |
+|  staple_image = staple_filter.Execute(image1,             |  staple_filter.SetForegroundValue(foregroundValue)      |
+|                                       image2, image3,     |  staple_filter.SetMaximumIterations(maximumIterations)  |
+|                                       confidenceWeight,   |  staple_image = staple_filter()                         |
+|                                       foregroundValue,    |                                                         |
```

Comment:
New code, line `staple_image = staple_filter()` should be changed to `staple_filter.Execute([image1, image2, image3])`